src/components/common/Actions.vue

```diff
@@ -24,7 +24,7 @@
     <div class="buttons-right">
       <v-fade-transition hide-on-leave>
         <v-btn id="file-pay-btn" large color="primary"
-          :disabled="!getHaveCorrection || isBusySaving"
+          :disabled="!isFilingChanged || isBusySaving || !isFilingValid || !isStaffPaymentValid"
```

Comment:
Why isn't "staff payment validity" simply part of the filing validity?

Reply:
I am c hecking the filing validity in save and resume. If that is not needed it can be part of the filing validity.

Reply:
What I mean is that "staff payment validity" should be part of the "is filing valid" getter.

Reply:
Done